acceptance/server_test.go

```diff
@@ -1,97 +1,117 @@
 package acceptance_test
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
 	"net/http"
 
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+
 	"github.com/databricks/cli/libs/testserver"
-	"github.com/databricks/databricks-sdk-go/service/catalog"
-	"github.com/databricks/databricks-sdk-go/service/compute"
-	"github.com/databricks/databricks-sdk-go/service/iam"
 	"github.com/databricks/databricks-sdk-go/service/workspace"
 )
 
 func AddHandlers(server *testserver.Server) {
-	server.Handle("GET /api/2.0/policies/clusters/list", func(r *http.Request) (any, int) {
-		return compute.ListPoliciesResponse{
-			Policies: []compute.Policy{
-				{
-					PolicyId: "5678",
-					Name:     "wrong-cluster-policy",
-				},
-				{
-					PolicyId: "9876",
-					Name:     "some-test-cluster-policy",
-				},
-			},
-		}, http.StatusOK
+	server.Handle("GET /api/2.0/policies/clusters/list", func(fakeWorkspace *testserver.FakeWorkspace, r *http.Request) (any, int) {
+		return fakeWorkspace.PoliciesList()
+	})
+
+	server.Handle("GET /api/2.0/instance-pools/list", func(fakeWorkspace *testserver.FakeWorkspace, r *http.Request) (any, int) {
+		return fakeWorkspace.InstancePoolsList()
 	})
 
-	server.Handle("GET /api/2.0/instance-pools/list", func(r *http.Request) (any, int) {
-		return compute.ListInstancePools{
-			InstancePools: []compute.InstancePoolAndStats{
-				{
-					InstancePoolName: "some-test-instance-pool",
-					InstancePoolId:   "1234",
-				},
-			},
-		}, http.StatusOK
+	server.Handle("GET /api/2.1/clusters/list", func(fakeWorkspace *testserver.FakeWorkspace, r *http.Request) (any, int) {
+		return fakeWorkspace.ClustersList()
```

Comment:
This refactoring seems unrelated. Could you restore original code? In general, I'd prefer to keep it simple and avoid extra indirections for no reason.

Reply:
The whole file becomes quite complex, especially because we will need to add many more handlers. I will follow suggestion from another thread and we can iterate separately.